addon/utils/article-structure-plugin/constants.ts

```diff
@@ -0,0 +1,260 @@
+import { StructureSpec } from '@lblod/ember-rdfa-editor-lblod-plugins/plugins/article-structure-plugin';
+import romanize from '../romanize';
+
+export const STRUCTURES: Record<string, StructureSpec> = {
+  title: {
+    uriBase: 'http://data.lblod.info/titles/',
+    title: 'Title',
+    type: 'https://say.data.gift/ns/Title',
+    numberPredicate: 'http://data.europa.eu/eli/ontology#number',
+    numberingFunction: romanize,
+    heading: 'h3',
+    translation: 'article-structure-plugin.insert.title',
+    moveUp: 'article-structure-plugin.moveUp.title',
+    moveDown: 'article-structure-plugin.moveDown.title',
+    shaclConstraint: `
+      @prefix sh: <http://www.w3.org/ns/shacl#> .
+      @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+      @prefix schema: <http://schema.org/> .
+      schema:ArticleShape
+        a sh:NodeShape  ;
+        sh:targetSubjectsOf <http://www.w3.org/1999/02/22-rdf-syntax-ns#type>;
+        sh:property [
+          sh:path <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> ;
+          sh:hasValue <https://say.data.gift/ns/DocumentContent>
+        ].
+      `,
+    template: (uri, intlService) => `
+    <div 
+      property="say:hasPart" 
+      typeof="https://say.data.gift/ns/Title https://say.data.gift/ns/ArticleContainer" 
+      resource="${uri}"
+    >
+      <h4 property="say:heading">
+        <span property="eli:number" datatype="xsd:string">
+          <span class="mark-highlight-manual">Voer inhoud in</span>
```

Comment:
We're inserting a textual placeholder where the number should go...

(same issue in all templates)

Reply:
disregard, misunderstood the logic, and I accidentally had local changes while testing